packages/shared/src/components/sidebar/ClickableNavItem.tsx

```diff
@@ -32,17 +37,20 @@ export function ClickableNavItem({
 
   if (!isButton && (!item.action || item.path)) {
     return (
-      <Link href={item.path} passHref prefetch={false}>
-        <a
-          {...(item.action && { onClick: item.action })}
-          {...props}
-          target={item?.target}
-          className={navBtnClass}
-          rel="noopener noreferrer"
-        >
-          {children}
-        </a>
-      </Link>
+      <>
+        <Link href={item.path} passHref prefetch={false}>
```

Comment:
It was already set before I just copied the code because of the added `Fragment`. But yeah `prefetch={false}` here is good because you probably don't want to prefetch all the menu links after user lands on the page. This still prefetches on link hover which is good but IMO bad DX from next team because this should (by naming) probably disable `prefetch` all together.